db/migrate/20150330233356_create_users.rb

```diff
@@ -8,6 +8,9 @@ def change
       t.string :profile_picture, :default => "default-profile-pic.png"
       t.string :favorite_games
       t.boolean :host
+      t.string :provider, :default => 'facebook'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.